01_fib/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
This is really good so far.
Should add some tests for the fib function directly, check output for a few edge cases (low, negative numbers, large numbers that trigger overflow).

Reply:
I think @anzboi means, write:
```go
func TestFib(t *testing.T) {
// test corner cases here
}